src/cloud/lib/i18n/enUS.ts

```diff
@@ -501,6 +503,16 @@ const enTranslation: TranslationSource = {
 
   [lngKeys.GeneralYes]: 'Yes',
   [lngKeys.GeneralNo]: 'No',
+
+  [lngKeys.ModalUnlockDashboardsTitle]: 'Unlock Dashboards',
+  [lngKeys.ModalUnlockDashboardsDescription]:
+    'Create unlimited dashboards to organize your docs, tasks.. in order to guide your workflow in your own way! Display the information the way you want to see it.',
```

Comment:
`your docs, tasks.. in order to ...` -> `your docs, tasks, etc., in order to ...`